File: app/src/main/java/com/example/singlescreenapp/MainActivity.java
package com.example.singlescreenapp;

import android.os.Build;
import android.support.annotation.RequiresApi;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import java.util.Objects;

public class MainActivity extends AppCompatActivity {

    @RequiresApi(api = Build.VERSION_CODES.KITKAT)
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        //Remove title bar
        Objects.requireNonNull(this.getSupportActionBar()).hide();

        setContentView(R.layout.activity_main);
    }
}

File: app/src/main/java/com/example/singlescreenapp/MainActivity.kt
package com.example.singlescreenapp

import android.os.Build
import android.os.Bundle
import androidx.annotation.RequiresApi
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {

    @RequiresApi(api = Build.VERSION_CODES.KITKAT)
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        //Remove title bar
        if (supportActionBar != null)
            supportActionBar?.hide()

        setContentView(R.layout.activity_main)
    }
}
